sdk/spring/spring-cloud-azure-starter-stream-eventhubs/pom.xml

```diff
@@ -45,7 +45,7 @@
   <!-- Repositories definitions -->
   <repositories>
     <repository>
-      <id>ossrh</id>
+      <id>repo-ossrh</id>
```

Comment:
We should think of pulling these repositories into a Spring parent POM, and some other common configurations between Azure Spring libraries